packages/next/src/client/components/react-dev-overlay/internal/helpers/stack-frame.ts

```diff
@@ -73,23 +68,23 @@ function getOriginalStackFrame(
       error: false,
       reason: null,
       external: true,
-      expanded: false,
       sourceStackFrame: source,
       originalStackFrame: null,
       originalCodeFrame: null,
       sourcePackage: null,
+      ignored: true,
     })
   }
 
   return _getOriginalStackFrame().catch((err: Error) => ({
     error: true,
     reason: err?.message ?? err?.toString() ?? 'Unknown Error',
     external: false,
-    expanded: false,
     sourceStackFrame: source,
     originalStackFrame: null,
     originalCodeFrame: null,
     sourcePackage: null,
+    ignored: true,
```

Comment:
blocking: We shouldn't hide frames that we can't sourcemap. Neither terminal nor Browser devtools work that way.